package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-autoql",
-  "version": "8.11.1",
+  "version": "8.11.2",
```

Comment:
Since this is a new feature the version should be 8.12.0

For reference: https://semver.org/